homeassistant/components/fritzbox/binary_sensor.py

```diff
@@ -60,6 +60,33 @@ class FritzBinarySensorEntityDescription(
         suitable=lambda device: device.device_lock is not None,
         is_on=lambda device: not device.device_lock,
     ),
+    FritzBinarySensorEntityDescription(
+        key="battery_low",
+        translation_key="battery_low",
+        device_class=BinarySensorDeviceClass.BATTERY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        suitable=lambda device: device.battery_low is not None,
+        is_on=lambda device: device.battery_low,
+        entity_registry_enabled_default=False,
+    ),
+    FritzBinarySensorEntityDescription(
+        key="holiday_active",
+        translation_key="holiday_active",
+        suitable=lambda device: device.holiday_active is not None,
+        is_on=lambda device: device.holiday_active,
+    ),
+    FritzBinarySensorEntityDescription(
+        key="summer_active",
+        translation_key="summer_active",
+        suitable=lambda device: device.summer_active is not None,
+        is_on=lambda device: device.summer_active,
+    ),
+    FritzBinarySensorEntityDescription(
```

Comment:
It is on purpose that `BinarySensorDeviceClass.WINDOW` hasn't been used here, as this doesn't represent a specific window to be open or not, rather than that the "window open dectection" on the thermostat has detected any window to be opened, so we can't ensure that in state "off" all windows are closed